opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -195,7 +195,9 @@ namespace Opm
             data::Wells res = WellState::report();
 
             const int nw = this->numWells();
-            const int np = this->numPhases();
+            // If there are now wells numPhases throws a floating point
+            // exception.
+            const int np = nw ? this->numPhases() : -1;
```

Comment:
Ok.

(But I don't see why we'd need an if, that was the point of putting it in the loop where it would unconditionally be ok?)